base/core/src/main/java/org/openscience/cdk/stereo/AbstractStereo.java

```diff
@@ -112,6 +112,21 @@ public void setConfigOrder(int cfg) {
         value = getConfigClass() | cfg;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public int getGroupInfo() {
+        return value & IStereoElement.GRP_MASK;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public void setGrpConfig(int grp) {
+        value &= ~IStereoElement.GRP_MASK;
+        value |= (grp & IStereoElement.GRP_MASK);
```

Comment:
I wish I could read this fluently :) (no change needed and not sure a comment of what it does would help much without understanding the use of bits in ints anyway)